<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZegoMainDialog</class>
 <widget class="QDialog" name="ZegoMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoTalk</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fafafa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="m_zoneTitle" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #0e88eb;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
	 font-size: 16px;
     color: #ffffff;</string>
            </property>
            <property name="text">
             <string>VideoTalk (国内版)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="m_bJumpToNet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:!hover{
    border: 1px solid #0e88eb;
    background-color: #0e88eb;
}

QPushButton:hover{
    border: 1px solid #0d80de;
    background-color: #0d80de;
}

QPushButton:hover:pressed{
    border: 1px solid #0d79d1;
    background-color: #0d79d1;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="style.qrc">
              <normaloff>:/Resources/images/official.png</normaloff>:/Resources/images/official.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_bMin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:!hover{
    border: 1px solid #0e88eb;
    background-color: #0e88eb;
}

QPushButton:hover{
    border: 1px solid #0d80de;
    background-color: #0d80de;
}

QPushButton:hover:pressed{
    border: 1px solid #0d79d1;
    background-color: #0d79d1;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="style.qrc">
              <normaloff>:/Resources/images/min.png</normaloff>:/Resources/images/min.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_bClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:!hover{
    border: 1px solid #0e88eb;
    background-color: #0e88eb;
}

QPushButton:hover{
    border: 1px solid #0d80de;
    background-color: #0d80de;
}

QPushButton:hover:pressed{
    border: 1px solid #0d79d1;
    background-color: #0d79d1;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="style.qrc">
              <normaloff>:/Resources/images/close.png</normaloff>:/Resources/images/close.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_lbTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family: 微软雅黑;
font-size: 30px;
color: #666666;</string>
         </property>
         <property name="text">
          <string>ZegoVideoTalk</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="m_edRoomID">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
     padding-left: 10px;
 }

 QLineEdit:!hover{
     border: 1px solid #e6e6e6;
     border-radius: 2px;
     font-family: 微软雅黑;
     font-size: 16px;
	 background-color: #fdfdfd;
	 color: #666666;
 }

 QLineEdit:hover:!focus{
     border: 1px solid #0e88eb;
     border-radius: 2px;
     font-family: 微软雅黑;
     font-size: 16px;
	 background-color: #fdfdfd;
	 color: #666666;
 }

 QLineEdit:focus{
     border: 1px solid #0e88eb;
     border-radius: 2px;
     font-family: 微软雅黑;
     font-size: 16px;
	 background-color: #ffffff;
 }</string>
           </property>
           <property name="placeholderText">
            <string>请输入房间号</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="m_bSettings">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:!hover{
     font-family: 微软雅黑;
     font-size: 16px;
	 border-radius: 2px;
     border: 1px solid #0e88eb;
     background-color: #ffffff;
     color: #0e88eb;
 }

 QPushButton:hover:!pressed{
     font-family: 微软雅黑;
     font-size: 16px;
	 border-radius: 2px;
     border: 1px solid #0e88eb;
     background-color: #0e88eb;
     color: #ffffff;
 }

 QPushButton:hover:pressed{
     font-family: 微软雅黑;
     font-size: 16px;
	 border-radius: 2px;
     border: 1px solid #0d80de;
     background-color: #0d80de;
     color: #ffffff;
 }</string>
             </property>
             <property name="text">
              <string>设置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_bEnterRoom">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:!hover{
     font-family: 微软雅黑;
     font-size: 16px;
	 border-radius: 2px;
     border: 1px solid #0e88eb;
     background-color: #ffffff;
     color: #0e88eb;
 }

 QPushButton:hover:!pressed{
     font-family: 微软雅黑;
     font-size: 16px;
	 border-radius: 2px;
     border: 1px solid #0e88eb;
     background-color: #0e88eb;
     color: #ffffff;
 }

 QPushButton:hover:pressed{
     font-family: 微软雅黑;
     font-size: 16px;
	 border-radius: 2px;
     border: 1px solid #0d80de;
     background-color: #0d80de;
     color: #ffffff;
 }</string>
             </property>
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZegoMainDialog.qrc"/>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
